docker-compose-native/prestissimo/workers/config_2.properties

```diff
@@ -0,0 +1,9 @@
+discovery.uri=http://172.19.0.4:8080
+http-server.http.port=8082
+presto.version=test
+system-memory-gb=3
```

Comment:
this is greater than specified in compose file.

Reply:
Updated, thanks.